addons/mail/models/mail_channel.py

```diff
@@ -826,6 +826,7 @@ def channel_info(self):
                 'public': channel.public,
                 'group_based_subscription': bool(channel.group_ids),
                 'create_uid': channel.create_uid.id,
+                'authorized_group': channel.group_public_id.full_name,
```

Comment:
"name"/"full_name" part should be present on the name of the key, unless we actually return the group on which case it should be a dict with id/full_name at least (or even better an insert command after creating the group model)

Also as this is a fake key only used in JS, directly use camelCase on the result too, in preparation to be able to get rid of convertData.